cvpaperchallenge-alumni/limit-lab-homepage
# Enhancement/prepare basic top page design

## Issue URL

close: #6

## Change overview

- Implement a rough layout and draft design.

## How to test

Check out the top page design, layout, and css animation

The page you will see should look something like this.

![image](https://github.com/user-attachments/assets/070e41c6-d15b-4ec3-8cd6-0251f2f3c346)

## Note for reviewers

> [!Important]
> This is just an initial draft, so there is a high possibility of changes. Therefore, the images for "Recent News" have been left as placeholders. (I’m considering making an interactive image combining FDSL-related papers in a stylish way.)
Similarly, responsive design has not been implemented for the same reason. (A task for this has been created as #7 .)
